### PR TITLE
chore(package): update chai to version 4.1.0

### PR DESCRIPTION
**Pull request by daedalus28**
Closes #91

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Please add a changelog entry for your changes.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Please add a changelog entry for your changes.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # 1.26.0
+Update chai to version 4.1.0
```
Comment (daedalus28) [inline CHANGELOG.md]: Need to bump the version here - we aren't releasing 1.26.0 twice :)

Comment (geosp) [inline CHANGELOG.md]: got it

**Pull request merged**